pl-csb-model/train/model_finetuner.py

```diff
@@ -57,7 +57,7 @@ def __train(self, model: AutoModelForSeq2SeqLM, tokenizer: NllbTokenizer, data:
         losses = []
         scheduler = get_constant_schedule_with_warmup(optimizer, num_warmup_steps=int(train_conf["warmup_steps"]))
 
-        LANGS = [("pl", "pol_Latn"), ("csb", "csb_Latn")]
```

Comment:
Removed hardcoded old column names since they are the same as language names now